content/blog/2022/11/airgap.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Installing and Running Kubewarden In Air-Gapped Environments
+authors:
+- Raul Cabello Martin
+date: 2022-11-28
+---
+
+We are glad to announce that deploying Kubewarden in air-gapped environments has been simplified and documented! For that, you will need a 
```

Comment:
nit picking: inside of the docs we used the term `air gap` instead of `air-gapped`. I think we should standardize on one term just to stay consistent

Reply:
done https://github.com/kubewarden/kubewarden.io/pull/153/commits/5a6476b4d1610a40ca086de4421ad0475af08597